Appraisals

```diff
@@ -369,6 +369,7 @@ elsif Gem::Version.new('2.2.0') <= Gem::Version.new(RUBY_VERSION) \
       gem 'sqlite3', '~> 1.3.6'
       gem 'sucker_punch'
       gem 'typhoeus'
+      gem 'que', '1.0.0.beta2'
```

Comment:
```suggestion
      gem 'que', '>= 1.0.0.beta2'
```
We like to keep these unlocked, as when the newest version gets released we are already testing against it.

Reply:
That's my bad, totally overlooked that bit. Fixed now. Thank you for catching all these things.